lib/puppetserver/ca/config/puppet.rb

```diff
@@ -148,6 +160,30 @@ def resolve_settings(overrides = {})
             settings[setting_name] = substitutions[substitution_name] = subbed_value
           end
 
+          configured_cadir = overrides.fetch(:cadir, false)
```

Comment:
Should probably pull this into its own method to clean up the caller and make testing easier.